Eduria/Eduria/Views/TimePeriods/Index.cshtml

```diff
@@ -1,23 +1,22 @@
 ﻿
 @{
-    ViewData["Title"] = "_ViewTimePeriods";
```

Comment:
Tijdlijn pijltjes kunnen mooier of misschien een melding dat je de tijdlijn kan slepen.

Reply:
Hoort in algemene layout